GUI/SearchTaskWindow.xaml

```diff
@@ -200,16 +354,36 @@
                                     </Grid.ColumnDefinitions>
                                     <StackPanel>
                                         <StackPanel Orientation="Horizontal" Margin="5">
-                                            <CheckBox x:Name="addCompIonCheckBox" Content="Generate Complementary Ions" />
+                                            <CheckBox x:Name="addCompIonCheckBox" Content="Generate Complementary Ions" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                                <CheckBox.ToolTip>
+                                                    <TextBlock>
+                                                        Generates experimental (not theoretical) complementary ions for each MS2.
+                                                        <LineBreak/>
+                                                        Useful for localization of modifications.
+                                                    </TextBlock>
+                                                </CheckBox.ToolTip>
+                                            </CheckBox>
                                         </StackPanel>
                                         <StackPanel Orientation="Horizontal" Margin="5">
                                             <Label x:Name="maxFragmentSize" Content="Max Fragment Mass (Da)" />
-                                            <TextBox x:Name="txtMaxFragmentSize" PreviewTextInput="PreviewIfInt" Width="45" IsEnabled="{Binding IsChecked, ElementName=classicSearchRadioButton, Converter={StaticResource InvertedBooleanConverter}}" />
+                                            <TextBox x:Name="txtMaxFragmentSize" PreviewTextInput="PreviewIfInt" Width="45" IsEnabled="{Binding IsChecked, ElementName=classicSearchRadioButton, Converter={StaticResource InvertedBooleanConverter}}" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                                <TextBox.ToolTip>
+                                                    <TextBlock>
+                                                        Indexed Search Only: Limits the maximum MS2 fragment mass to improve RAM efficiency.
```

Comment:
Indexed Search Only: Limits the maximum MS2 fragment mass to avoid rare crashes. A very high value (e.g. 30000) does not impact performance.